### PR TITLE
Inventory update & usage/comments added to initial.sh

### PR DESCRIPTION
**Pull request by thomasbeaudry**

### CODE REVIEW

File: bin/initial.sh [hunk 1]
```diff
@@ -1,5 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (gdevenyi) [inline bin/initial.sh:1]: we don't usually use a bash other than the system one on /bin/bash

File: bin/initial.sh [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+# The following script is run when deploying / upgrading a machine. It should be run from the ansible-playbook directory.
+# Ensure that the hostname that you run this on has already been added to the inventory file.
+
+if [ "$#" -ne 1 ]; then
```
Comment (gdevenyi) [inline bin/initial.sh:5]: Can you run shellcheck on this file, I think this probably should be "$@"

File: bin/initial.sh [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+  echo "USAGE: bin/initial.sh <hostname>"
+  exit 1
+fi
 set -euo pipefail
+
 ansible-playbook -K -k -v -i inventory initial-setup.yml --limit $1
```
Comment (gdevenyi) [inline bin/initial.sh:11]: Might as well do
hostnames=$1 and use that everywhere

**Review by gdevenyi — CHANGES_REQUESTED** [review]

File: bin/initial.sh (hunk 1 above)
Comment (thomasbeaudry) [reply bin/initial.sh:1]: Yes I know, but it's a public repo so I thought we should keep it universal incase people use it as inspiration for their own setups. Also if we ever deploy on a different OS, it's one less future bug

File: bin/initial.sh (hunk 2 above)
Comment (thomasbeaudry) [reply bin/initial.sh:5]: Is the intention to use this script on multiple hostnames at the same time?  $@ gives the argument as an array (which is good for multiple input arguments), $# is the number of arguments given to the script, so if it gets a second argument it fails

Comment (gdevenyi) [reply bin/initial.sh:5]: Ah right, sorry, I thought that was "$*" which is the whole command line.

File: bin/initial.sh (hunk 1 above)
Comment (gdevenyi) [reply bin/initial.sh:1]: Surprised to find `/usr/bin/env` is in-fact a POSIX standard location, so this is indeed more correct.

**Review by gdevenyi — APPROVED** [review]

**Pull request merged**